renku_notebooks/api/schemas.py

```diff
@@ -553,3 +544,23 @@ class AutosavesList(Schema):
 
     pvsSupport = fields.Bool(required=True)
     autosaves = fields.List(fields.Nested(AutosavesItem), missing=[])
+
+
+class ErrorResponseNested(Schema):
+    """Generic error response that can be used and displayed by the UI.
```

Comment:
What's the reasoning behind introducing this mapping instead of just using the matching http error codes?

Reply:
In the design of the error response schema the UI wanted to have some kind of a code. So I added this.